ext/afform/core/Civi/Api4/Action/CustomGroup/GetAfforms.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Civi\Api4\Action\CustomGroup;
+
+use CRM_Afform_ExtensionUtil as E;
+
+/**
+ * Get dynamic afforms for a custom group.
+ *
+ * For single value custom groups we generate:
+ * - a field block with all of the fields from the group
+ * - a submission form for editing these fields on a parent entity record
```

Comment:
Not sure how this would work. Probably not needed unless I'm missing something?

Reply:
Agreed, the multi-record custom groups will need to be treated as primary entities.